src/utils/auth/state/userState.js

```diff
@@ -47,7 +47,7 @@ class UserState {
     if (this.getIsSignedIn()) {
       const payload = jwtUtils.getTokenPayload(tokenService.getAccessToken());
       if (payload !== null) {
-        return payload.adminLevel;
+        return payload.privilegeLevel;
```

Comment:
This broke a test, what is it doing?

Reply:
adminlevel is not a field in payload. PrivilidgeLevel is what its called now. Ex. user = 0, admin = 2. The test most likely needs to be updated. my bbbb